test/libsolidity/semanticTests/isoltestTesting/effects.sol

```diff
@@ -0,0 +1,14 @@
+contract SmokeTest {
```

Comment:
Oh, I did not know we had a directory for isoltest tests (`isoltestTesting/`). Now that I see it, I think the tests from the external source PR should have been placed here.

Reply:
I think it was created afterwards.